docs/developers_guide/ocean/test_groups/global_ocean.rst

```diff
@@ -454,6 +436,82 @@ The default config options for this mesh are:
 The vertical grid is a ``60layerPHC`` profile (see :ref:`dev_ocean_framework_vertical`)
 with 60 vertical levels ranging in thickness from 10 to 250 m.
 
+.. _dev_ocean_global_ocean_kuroshio:
+
+Kuroshio8to60 and Kuroshio12to60
+++++++++++++++++++++++++++++++++
+
+The ``Kuroshio8to60`` and ``Kuroshio12to60`` mehses are designed to explore
+dynamics of the Kuroshio Current.
```

Comment:
Kuroshio Current -> ''Western Boundary Current (WBC) in the North Pacific Ocean, the Kuroshio.

Reply:
```suggestion
dynamics of the Western Boundary Current (WBC) in the North Pacific Ocean, the Kuroshio.
```